vnext/Microsoft.ReactNative/Modules/WebSocketModuleUwp.cpp

```diff
@@ -112,10 +111,13 @@ void LegacyWebSocketModule::WebSocket::connect(
       std::string response;
 
       // Use ABI to avoid throwing exceptions on standard errors
-      winrt::com_ptr<ABI::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs> abiArgs{
-          args.as<ABI::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs>()};
-      winrt::com_ptr<ABI::Windows::Storage::Streams::IDataReader> abiReader;
-      auto hr = abiArgs->GetDataReader(abiReader.put());
+      auto abiArgs = reinterpret_cast<
+          winrt::impl::abi_t<winrt::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs> *>(
+          winrt::get_abi(
+              static_cast<winrt::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs const &>(
```

Comment:
why this raw cast instead of continuing to use `.as()` ?

Reply:
same in PackagerConnection.cpp - worth adding a comment any time we drop to abi about why

Reply:
There is a comment above saying that we "Use ABI to avoid throwing exceptions on standard errors". 
What other info would you suggest to add?

In this code we use the ABI types using the C++/WinRT way and to avoid referencing the WinSDK WRL-based definitions as we had before because they required to bring a lot of extra header files.

We do not use .as() because we do not need to run QueryInterface - we just want to access the underlying pointer.